docs/info_for_validators.md

```diff
@@ -0,0 +1,31 @@
+# Info for Validators
+
+Validators may want to read this document.
+
+##  Key Assignment
+
+The key assignment feature enables validators to use different consensus keys for each consumer chain. The feature allows validators to submit an AssignConsumerKey transaction to the provider chain. Pseudocode for type:
+
+```txt
+message MsgAssignConsumerKey {
+  // The chain id of the consumer chain to assign a consensus public key to
+  string chain_id;
+  // The validator address on the provider
```

Comment:
Operator addr or cons addr? See https://github.com/cosmos/interchain-security/pull/672#discussion_r1084325871.

We should consider renaming/documenting this field accordingly

Reply:
cc @mpoke

Reply:
Operator address. See https://github.com/cosmos/interchain-security/blob/b9d62995662a86c5b8eeb053c245ecfbd801458f/x/ccv/provider/keeper/msg_server.go#L44